### ISSUE TITLE
Fluid and softbody

### ISSUE THREAD

**Issue by dbokis** [DanielChappuis/reactphysics3d#118]
I admire your work on this project and I would very much like to suggest some additional implementations that would be interesting to see . Fluid and soft body mesh simulations are sometimes extremely complicated in order to run on real time so if you could add them I would be more than thankful to you .

P.S. scaling matrices does not work ( for the collision shapes of rigid bodies ) .

**Comment by DanielChappuis** [comment]
Thanks a lot for your feedback.

Of course I plan to add support for soft bodies and fluid (in that order) in the future.

Can I ask you to open a new issue for the scaling matrices and describe what is not working exactly ?

About the documentation, the online documentation is always supposed to be up to date with the latest released version of the library. If you think something is not correct in the current documentation, could you please open another issue about it.

Thanks a lot.

**Comment by zeunio** [comment]
Fluid support would be awesome, when are you planning on adding it?

**Comment by DanielChappuis** [comment]
Yes it's in my mind but really not anytime soon. I have so much things I want to do first (multi-threading, continuous collision detection, soft bodies, ...).
